extensions/amp-install-serviceworker/0.1/test/test-amp-install-serviceworker.js

```diff
@@ -86,7 +86,7 @@ describes.realWin('amp-install-serviceworker', {
   it('should be ok without service worker.', () => {
     const install = doc.createElement('amp-install-serviceworker');
     const implementation = install.implementation_;
-    expect(implementation).to.be.defined;
+    expect(implementation).to.not.be.undefined;
```

Comment:
We should replace `to.not.be.undefined` with `to.exist` since `null` will pass.

http://chaijs.com/api/bdd/#method_exist

Reply:
Good catch. Fixed.